gems/railties/6.0/_scripts/test

```diff
@@ -19,3 +19,4 @@ bundle exec rbs --repo $REPO_DIR -r tsort validate --silent
 # bundle exec steep check
 #
 # $(git rev-parse --show-toplevel)/bin/check-untyped-call.rb
+$(git rev-parse --show-toplevel)/bin/check-manifest-yaml.rb
```

Comment:
I added this line after posting this PR. This file was added at https://github.com/ruby/gem_rbs_collection/pull/417